File: cardbattlemodule/src/main/java/com/syahputrareno975/cardbattlemodule/model/player/PlayerModel.java
package com.syahputrareno975.cardbattlemodule.model.player;

import java.io.Serializable;

public class PlayerModel implements Serializable {
    public String Id = "";
    public String  Name = "";
    public String Avatar = "";
    public int Level = 0;
    public Long Cash = 0L;
    public Long Exp  = 0L;
    public Long MaxExp = 0L;
    public int MaxReserveSlot = 0;
    public int MaxDeckSlot = 0;

    public PlayerModel() {
    }

    public PlayerModel(String id, String name, String avatar, int level, Long cash, Long exp, Long maxExp, int maxReserveSlot, int maxDeckSlot) {
        Id = id;
        Name = name;
        Avatar = avatar;
        Level = level;
        Cash = cash;
        Exp = exp;
        MaxExp = maxExp;
        MaxReserveSlot = maxReserveSlot;
        MaxDeckSlot = maxDeckSlot;
    }
}

File: app/src/main/java/com/syahputrareno975/simplecardbattle/model/player/PlayerModel.kt
package com.syahputrareno975.simplecardbattle.model.player

import java.io.Serializable

class PlayerModel : Serializable {
    var Id = ""
    var Name = ""
    var Avatar = ""
    var Level = 0
    var Cash: Long = 0
    var Exp : Long = 0
    var MaxExp : Long = 0
    var MaxReserveSlot  = 0
    var MaxDeckSlot = 0

    constructor()
    constructor(
        Id: String,
        Name: String,
        Avatar: String,
        Level: Int,
        Cash: Long,
        Exp: Long,
        MaxExp: Long,
        MaxReserveSlot: Int,
        MaxDeckSlot: Int
    ) {
        this.Id = Id
        this.Name = Name
        this.Avatar = Avatar
        this.Level = Level
        this.Cash = Cash
        this.Exp = Exp
        this.MaxExp = MaxExp
        this.MaxReserveSlot = MaxReserveSlot
        this.MaxDeckSlot = MaxDeckSlot
    }


}